src/shared/ui/surfaces/Card/Card.tsx

```diff
@@ -0,0 +1,52 @@
+import Image from "next/image";
+import { IoMdPlay } from "react-icons/io";
+import { twJoin } from "tailwind-merge";
+
+import { IconButton } from "../../inputs";
+
+import { Paper } from "..";
```

Comment:
Could you update the update the import to be more explicit here? It should be `../Paper/Paper` I think.

(importing from "/surfaces/index.ts" is reserved for components outside of the `surfaces` folder

Reply:
Changed !